sdk/loadtestservice/Azure.Analytics.LoadTestService/README.md

```diff
@@ -0,0 +1,90 @@
+# Azure LoadTestService client library for .NET
+
+This section should give out brief introduction of the client library.
+
+* First sentence: **Describe the service** briefly. You can usually use the first line of the service's docs landing page for this (Example: [Cosmos DB docs landing page](https://docs.microsoft.com/azure/cosmos-db/)).
+* Next, add a **bulleted list** of the **most common tasks** supported by the package or library, prefaced with "Use the client library for [Product Name] to:". Then, provide code snippets for these tasks in the [Examples](#examples) section later in the document. Keep the task list short but include those tasks most developers need to perform with your package.
+
+  [Source code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/loadtestservice/Azure.Analytics.LoadTestService/src) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Analytics.LoadTestService) | [API reference documentation](https://azure.github.io/azure-sdk-for-net) | [Product documentation](https://docs.microsoft.com/azure)
```

Comment:
@annelo-msft Do we have any release tag? I don't think we should point user to the latest code.